requirements.txt

```
@@ -1,5 +1,6 @@
-imjoy-rpc>=0.2.56
-python-socketio[asyncio_client]
-pyyaml
-aiohttp
-aiohttp_cors
+imjoy-rpc==0.2.72
```

Is this imjoy-rpc version compatible with master branch?

---

Let's keep this for now at 0.2.56. It'll be bumped in another PR.
```suggestion
imjoy-rpc==0.2.56
```